candi/bashible/common-steps/node-group/068_configure_kubelet_systemd_unit.sh.tpl

```diff
@@ -72,10 +72,12 @@ ExecStart=/usr/local/bin/d8-kubelet-forker /usr/bin/kubelet \\
     --node-labels=node.deckhouse.io/type={{ .nodeGroup.nodeType }} \\
     --bootstrap-kubeconfig=/etc/kubernetes/bootstrap-kubelet.conf \\
     --config=/var/lib/kubelet/config.yaml \\
+{{- if semverCompare "<1.24" .kubernetesVersion }}
```

Comment:
What is the substitution for these flags in Kubernetes 1.24?

Reply:
These flags were removed from kubelet in 1.24, they were used for dockershim, which was removed. Containerd and cri-o use CNI plugin by default, also conf dirs are directly passed to the containerd config, like:
```
cat /etc/containerd/config.toml  | grep cni
    [plugins."io.containerd.grpc.v1.cri".cni]
      bin_dir = "/opt/cni/bin"
      conf_dir = "/etc/cni/net.d"
```